content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -80,9 +80,34 @@ To account for modern web applications, loading time watches for network request
 - **Initial Load**: Loading Time is equal to _whichever is longer_:
 
   - The difference between `navigationStart` and `loadEventEnd`.
-  - Or the difference between `navigationStart` and the first time the page has no activity for more than 100ms (activity defined as ongoing network requests or a DOM mutation).
+  - Or the difference between `navigationStart` and the first time the page has no activity (see [next section][18] for details).
 
-- **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity for more than 100ms (activity defined as ongoing network requests or a DOM mutation).
+- **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity (see [next section][18] for details).
+
+### How is page activity calculated?
+
+Whenever a navigation or a click occurs, we start tracking the page activity to estimate the time until the interface is stable again. We consider that a page has activity by looking at network requests and DOM mutations. The page activity stops when there is no ongoing requests and no DOM mutation for more than 100ms. We consider that there was no activity if no requests or DOM mutation occurred in 100ms.
+
+This approach works fine most of the time, but sometimes requests made by the application do not reflect actual activity as they don't have impact on the interface. In particular, we identified two problematic use cases:
```

Comment:
The criteria of 100ms since last request or DOM mutation might not be an accurate determination of activity in the following scenarios: